browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2208,11 +2208,23 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		if (!name)
 			return '';
 
+		var iconURLAliases = {
+			'AlignLeft': ['LeftPara'],
```

Comment:
we can have only one alias: 'AlignLeft': ['LeftPara'] ->  'AlignLeft': 'LeftPara'